chapters/arrays.tex

```diff
@@ -1121,26 +1099,25 @@ \subsection{Array Element-wise Division}\doublelabel{array-element-wise-division
 
 \end{longtable}
 
-{[}\emph{Element-wise division by scalar (./) and division by scalar (/)
-are identical: a./s = a/s.}
-
-\emph{Example:}
-
+\begin{example}
+Element-wise division by scalar (\lstinline!./!) and division by scalar (\lstinline!/!)
+are identical: \lstinline!a./s = a/s!:
 \begin{lstlisting}[language=modelica]
   2./[1,2;3,4] // error, since 2.0/[1,2;3,4]
   2 ./[1,2;3,4] // fine, element-wise division
 \end{lstlisting}
-\emph{This is a consequence of the parsing rules, since 2. is a lexical
-unit. Using a space after the literal solves the problem.}{]}
+This is a consequence of the parsing rules, since `\lstinline!2!' is a lexical
+unit. Using a space after the literal solves the problem.
+\end{example}
 
 \subsection{Exponentiation of Scalars of Numeric Elements}\doublelabel{exponentiation-of-scalars-of-numeric-elements}
 
-Exponentiation "\lstinline!a^b!" is defined as pow(double a,double b) in the ANSI
-C library if both "\lstinline!a!" and "\lstinline!b!" are Real scalars. A Real scalar value is
-returned. If "a" or "b" are Integer scalars, they are automatically
-promoted to "Real". Consequences of exceptional situations, such as
-(a==0.0 \textbf{and} b\textless{}=0.0, a\textless{}0 \textbf{and} b is
-not an integer) or overflow are undefined
+Exponentiation \lstinline!a^b! is defined as \lstinline!pow(double a, double b)! in the ANSI
+C library if both \lstinline!a! and \lstinline!b! are \lstinline!Real! scalars. A \lstinline!Real! scalar value is
+returned. If \lstinline!a! or \lstinline!b! are \lstinline!Integer! scalars, they are automatically
+promoted to \lstinline!Real!. Consequences of exceptional situations, such as
+($\text{\lstinline!a!} = 0.0$ and $\text{\lstinline!b!} \leq 0.0$, $\text{\lstinline!a!} < 0$ and \lstinline!b! is
+not an \lstinline!Integer!) or overflow are undefined.
```

Comment:
Keep as integer, since pow(-3.0, 2.0) is defined in C, i.e. it means value that corresponds to an integer, not an Integer type.

Reply:
Good catch.  Done.